packages/sanity/src/core/versions/components/VersionBadge.tsx

```diff
@@ -0,0 +1,16 @@
+import {Box, Button} from '@sanity/ui'
+
+import {type Version} from '../types'
+import {VersionIcon} from './VersionIcon'
+
+export function VersionBadge(props: {version: Version}): JSX.Element {
+  const {version} = props
+
+  return (
+    <Box flex="none">
+      <Button as="a" mode="bleed" padding={0} radius="full">
```

Comment:
NIT:  There is a small difference in the hover behaviour comparing the globalBundleMenu and this button.
I think it's possible caused by the `as=a` property.
See how the gray background goes beyond the limits of the Badge and the underline of the text.

<img width="180" alt="Screenshot 2024-07-01 at 10 19 48" src="https://github.com/sanity-io/sanity/assets/46196328/3b5ed70b-5e75-4e5e-a990-a41b49418d0e">
<img width="185" alt="Screenshot 2024-07-01 at 10 19 52" src="https://github.com/sanity-io/sanity/assets/46196328/5de2beeb-c567-4b64-ab10-cb22b74d73c7">

Reply:
Quite eagle eyes 😄 
I had to change the brightness of my screen to catch that on your screenshots!
On my way to fix it 👍